### PR TITLE
code formatting change

### PR DESCRIPTION
**Pull request by wijith7**

### CODE REVIEW

File: .idea/checkstyle-idea.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (madurangasiriwardena) [inline .idea/checkstyle-idea.xml:1]: Why are these files added? These are local IDE configs.

Comment (wijith7) [reply .idea/checkstyle-idea.xml:1]: fixed

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# product-apim-training
\ No newline at end of file
+# product-apim-trainin
```
Comment (bhathiya) [inline README.md]: typo

**Review by bhathiya — APPROVED** [review]
